webpack.config.js

```diff
@@ -0,0 +1,62 @@
+const defaultConfig = require('@wordpress/scripts/config/webpack.config');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Probably I am lacking context here, but could you explain why we need to disable this eslint rule here? thanks!

Reply:
No, i don't think there was a reason it was there. Probably left over from the config this one was based on.